WEB-411: Improve alignment of Debit and Credit Amount fields

AnvayKharb opened the pull request:
On the Create Journal Entry page, the alignment of the Debit Amount and Credit Amount fields is inconsistent. These fields leave unnecessary empty space and are not aligned with the corresponding GL Entry fields, which affects the visual balance of the form.

This update adjusts only the alignment of these two fields so they fill the vacant space appropriately. No other UI elements or layout sections were changed.

### What Was Improved

Aligned Debit Amount field next to “Affected GL Entry (Debit)” by removing the empty gap.

Aligned Credit Amount field next to “Affected GL Entry (Credit)” in the same way.

Ensured no other form layout, spacing, or styling was modified.
### Related Issues and Discussion

Fixes: #WEB-411

### Before:
<img width="1336" height="283" alt="Screenshot 2025-11-05 at 11 57 09 PM" src="https://github.com/user-attachments/assets/d700af1f-8981-45cd-89cf-8fdf6f6c6011" />



### After:

<img width="1278" height="225" alt="Screenshot 2025-11-06 at 12 37 06 AM" src="https://github.com/user-attachments/assets/ea5a5932-4128-4fb7-9ee4-db3f5b1709e3" />



<!-- This is an auto-generated comment: release notes by coderabbit.ai -->
## Summary by CodeRabbit

* **Style**
  * Made Debit and Credit amount fields more compact and better aligned for improved form layout and easier data entry.

* **New Features**
  * Added an explicit add-entry button for the first row in both Debit and Credit sections; remove-entry controls remain available on subsequent rows.
<!-- end of auto-generated comment: release notes by coderabbit.ai -->

AnvayKharb commented:
heyy @gkbishnoi07 please consider

steinwinde reviewed (changes requested):
Please provide a single commit instead of two, i.e. squash your commits.

AnvayKharb committed:
WEB-411: Improve alignment of Debit and Credit Amount fields in Create Journal Entry form

- Changed Debit Amount field from flex-43 to flex-24 to align under Currency field
- Changed Credit Amount field from flex-43 to flex-24 to align under External Asset Owner field
- Kept all add/remove button functionality intact

AnvayKharb commented:
@steinwinde just updated my pr !

AnvayKharb commented:
@steinwinde you can check now

AnvayKharb commented:
@steinwinde @gkbishnoi07  please take a look

gkbishnoi07 commented:
@AnvayKharb please create or mention jira ticket related to this PRs

AnvayKharb commented:
heyy @gkbishnoi07 my jira ticket number is web-411 as I've mentioned

gkbishnoi07 commented:
Please mention in Pr description! Where it mentioned??

AnvayKharb commented:
Hey @gkbishnoi07 , that was my mistake — I hadn’t mentioned it earlier. I’ve now updated my PR description accordingly.

gkbishnoi07 commented:
@AnvayKharb this change doesn’t look good to me. What exactly did you modify? It seems like you’ve just added more empty space instead of fixing the issue.

gkbishnoi07 reviewed (changes requested):
if you want to make it good you can align the fields properly
“Affected GL Entry (Debit)” and “Debit Amount” should be placed directly under “Office”, and “Affected GL Entry (Credit)” and “Credit Amount” should be under “Currency” and “External Asset Owner” respectively.

gkbishnoi07 commented:
@AnvayKharb please update screenshot
